platform-core-module/src/main/java/com/softicar/platform/core/module/email/buffer/BufferedEmailSendProgram.java

```diff
@@ -42,6 +42,7 @@ public void sendForAllServers() {
 
 			AGBufferedEmail.TABLE//
```

Comment:
who has written this code? shouldn't this start with AGBufferedEmailServer.TABLE.createSelect()?????

Reply:
I think this was requested.
Only check servers where BufferedEmails exists.
Dunno when and why

Reply:
I can rewrite it a bit